packages/create-block/lib/templates.js

```diff
@@ -65,27 +72,41 @@ const templates = {
 	},
 };
 
-const getTemplate = ( templateName ) => {
-	if ( ! templates[ templateName ] ) {
+const getTemplate = async ( templateName ) => {
+
+	if ( await isCoreTemplate( templateName ) ) {
+		return templates[ templateName ];
+	}
+
+	// throw a CLIError if the the template is neither a core one nor an external one
+	if ( ! await isExternalTemplate( templateName ) && ! await isCoreTemplate( templateName ) ) {
```

Comment:
`await isCoreTemplate( templateName )` is called 5 lines above as well.

Reply:
Good catch. I refactored it so it doesn't call it twice  👍